spr/src/revision_utils.rs

```diff
@@ -5,32 +5,46 @@
 use crate::error::{Error, Result};
 
 /// Parse revision parameter and determine if it should be treated as a range
-/// Returns (use_range_mode, base_rev, target_rev)
+/// Returns (use_range_mode, base_rev, target_rev, is_inclusive)
+/// where is_inclusive indicates whether :: (inclusive) or .. (exclusive) operator was used
 pub fn parse_revision_and_range(
     revision_opt: Option<&str>,
     all_mode: bool,
     base_opt: Option<&str>,
-) -> Result<(bool, String, String)> {
+) -> Result<(bool, String, String, bool)> {
```

Comment:
Just a suggestion: Maybe it's worth creating a struct to return so the values are named. Maybe `RevisionRange`?

```rs
struct RevisionRange {
  use_range_mode: bool,
  base_rev: String,
  target_rev: String,
  is_inclusive: bool,
}
```